posixlib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -393,4 +393,21 @@ object pthread {
   @name("scalanative_pthread_mutexattr_t_size")
   def pthread_mutexattr_t_size: CSize = extern
 
+  @name("scalanative_sched_other")
+  def SCHED_OTHER: CInt = extern
+
+  @name("scalanative_sched_fifo")
+  def SCHED_FIFO: CInt = extern
+
+  @name("scalanative_sched_rr")
+  def SCHED_RR: CInt = extern
+
+  @name("scalanative_sched_batch")
+  def SCHED_BATCH: CInt = extern
+
+  @name("scalanative_sched_idle")
+  def SCHED_IDLE: CInt = extern
+
+  @name("scalanative_sched_deadline")
+  def SCHED_DEADLINE: CInt = extern
```

Comment:
The non-POSIX values should be documented, especially since they can lead to runtime failures https://stackoverflow.com/questions/43167963/setting-scheduling-policy-sched-idle-or-sched-batch-fails-with-einval-22

A simple scaladoc comment `/** Not defined in POSIX standard, might lead to runtime errors */`  for each of them should be enough

Reply:
Fair enough. I also moved this constant into the rigth place, near `sched_get_priority_max` where it should be.